CrossMod/EnvironmentModule.cs

```diff
@@ -10,32 +10,27 @@ namespace MagicStorage {
 	/// <summary>
 	/// A module of information for use in a Storage Configuration Interface. Only one instance is assumed to be active at once.
 	/// </summary>
-	public abstract class EnvironmentModule : ModType, ILocalizedModType
```

Comment:
`ILocalizedModType` usage should not have been removed.  please use the existing code in MagicStorage/1.4.4 as a comparison